modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/drag-and-drop/dragTypes.es.js

```diff
@@ -12,6 +12,10 @@
  * details.
  */
 
-export const DRAG_DATA_DEFINITION_FIELD = 'dataDefinitionField';
-export const DRAG_FIELD_TYPE = 'fieldType';
-export const DRAG_FIELDSET = 'fieldset';
+export const DRAG_DATA_DEFINITION_FIELD_MOVE = 'dataDefinitionField:move';
+export const DRAG_FIELD_TYPE_MOVE = 'fieldType:move';
+export const DRAG_FIELDSET_MOVE = 'fieldset:move';
```

Comment:
There must be no `DRAG_DATA_DEFINITION_FIELD_MOVE` and `DRAG_FIELDSET_MOVE` because they are all `DRAG_FIELD_TYPE_MOVE`, there is no difference when moving.

Reply:
It was previously removed! Thanks for reviewing Matu!